x/wasm/types/keys.go

```diff
@@ -60,6 +63,23 @@ func GetContractStorePrefix(addr sdk.AccAddress) []byte {
 	return append(ContractStorePrefix, addr...)
 }
 
+// GetAsyncPacketKey returns the key for a packet that is acknowledged asynchronously
+func GetAsyncPacketKey(destChannel string, sequence uint64) []byte {
+	// key is a concatenation of length-prefixed destination channel and sequence
+	channel := []byte(destChannel)
+	channelLen := make([]byte, 4)
+	binary.LittleEndian.PutUint32(channelLen, uint32(len(channel)))
+	seq := make([]byte, 8)
+	binary.LittleEndian.PutUint64(seq, sequence)
```

Comment:
Any reason for using LE instead of BE?

If we want to iterate though all packets of a given (port_id, destinationChannel) this would give us a strange order

Reply:
Nope, no reason other than all the example code I found being little endian. Will change to big endian